### PR TITLE
Update Add.php to fix #23990

### LINKED ISSUE #23990: Reset not working properly in Add review from admin
### Preconditions (*)

1. magento 2.3

### Steps to reproduce (*)

1. Go to Admin
2. Marketing -> All Reviews -> New Review
3. Click any product
4. Fill all data
5. Click on Reset

### Expected result (*)

1. Review form should reset and stay on same page
2. 

### Actual result (*)

1. https://www.screencast.com/t/dT6oqMKkF

### PR DESCRIPTION
**Pull request by arushibansal013**
Removed Magento existing reset button and added again so that on click value can be changed

### Description (*)
Under Marketing -> All Reviews -> New Review reset button was not working. Thus removed default reset and added again with new onClick functionality. As default reset will not work with form loaded via ajax, it will refresh the page.

### Fixed Issues (if relevant)

1. magento/magento2#23990: Reset not working properly in Add review from admin

### Manual testing scenarios (*)

1. Ui Level testing, click on reset button and test if form fields are getting reset.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by kanthy-i95dev — APPROVED** [review]
This is looking good

**Comment by arushibansal013** [comment]
@magento-engcom-team For above pull request database build failed but my changes are not related to that. In status, it's showing all as a success only. How can i know why it failed.

File: app/code/Magento/Review/Block/Adminhtml/Add.php [hunk 1]
```diff
@@ -21,16 +21,16 @@ class Add extends \Magento\Backend\Block\Widget\Form\Container
...
-
         $this->buttonList->update('save', 'label', __('Save Review'));
         $this->buttonList->update('save', 'id', 'save_button');
-
-        $this->buttonList->update('reset', 'id', 'reset_button');
-
+        $this->buttonList->remove('reset');
+        $this->buttonList->add(
+            'reset_button',
+            ['label' => 'Reset', 'onclick'=>"jQuery('#edit_form').trigger('reset')"]
```
Comment (VladimirZaets) [inline app/code/Magento/Review/Block/Adminhtml/Add.php]: Hi @arushibansal013. Thanks for collaboration. 
It isn't correct to pass JS code from PHP code. Also, you hardcode HTML selector directly in php code. 
Please move JS code to phtml template and use x-magento-init to inject it.

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Please view the comments. Thanks

File: app/code/Magento/Review/Block/Adminhtml/Add.php (hunk 1 above)
Comment (arushibansal013) [reply app/code/Magento/Review/Block/Adminhtml/Add.php]: Hi @VladimirZaets,

Can you review it now.

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-5583](https://jira.corp.magento.com/browse/ENGCOM-5583) has been created to process this Pull Request
:eight_spoked_asterisk: @VladimirZaets, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
Hi @arushibansal013 !

During testing we faced an issue.

_**Problem:**_  "Product Rating" block is not cleared after pressing the "Reset" button

**Actual Result:**
screen

@arushibansal013 Could you take a look, please?

Thanks!

**Comment by arushibansal013** [comment]
Hi @engcom-Alfa ,

Sure, I will check and update soon.

**Comment by VladimirZaets** [comment]
@arushibansal013, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**